scripts/monitor/monitor.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+
+# strict mode
+set -euo pipefail
+IFS=$'\n\t'
+
+# configurable vars
+INSOLAR_SCRIPTS_DIR=${INSOLAR_ARTIFACTS_DIR:-"scripts"}/
+LAUNCHNET_MONITOR_DIR=${LAUNCHNET_BASE_DIR:-"${INSOLAR_SCRIPTS_DIR}monitor"}/
+
+# Used by docker-compose config. DO NOT REMOVE.
+PROMETHEUS_IN_CONFIG=${LAUNCHNET_MONITOR_DIR}prometheus/prometheus.yaml
+
+set +x
+
+export PROMETHEUS_CONFIG_DIR=../../${LAUNCHNET_MONITOR_DIR}prometheus/
+
+if [[ $# -lt 1 ]]; then
+# 1) if started without params  pretend to be clever and do what is expected:
+# * prepare configuration
+# * shutdown ans start all monitoring services
+# * wait until Jaeger starts (we want to be sure tracer works before start benchmark)
```

Comment:
Is Jaeger used here? Maybe you should remove the line?

Reply:
No.remove this line